ntbs-service/Models/Factories/RiskFactorFactory.cs

```diff
@@ -0,0 +1,52 @@
+using ntbs_service.Models.Enums;
+
+namespace ntbs_service.Models.Factories
+{
+    public static class RiskFactorFactory
+    {
+        public static RiskFactorDetails RiskFactorDrugs() {
+            return new RiskFactorBuilder(RiskFactorType.Drugs).Build();
+        }
+
+        public static RiskFactorDetails RiskFactorHomelessness() {
+            return new RiskFactorBuilder(RiskFactorType.Homelessness).Build();
+        }
+
+        public static RiskFactorDetails RiskFactorImprisonment() {
+            return new RiskFactorBuilder(RiskFactorType.Imprisonment).Build();
+        }
+    }
+
+    public class RiskFactorBuilder
+    {

```

Comment:
This went beyond what you asked for when you mentioned a static factory method but I thought a builder like this might be quite useful for tests, what do you think? Makes it a bit more readable.

Reply:
I quite like builders as I do find they make tests a bit more readable (but as you say it's not a big difference), but on the other hand we'd want to be consistent and add them for every entity type which adds a lot of boilerplate, so happy to stick with the original way!